manager.go

```diff
@@ -209,7 +209,7 @@ func (m *Manager) SubsystemHealthChecks(ctx context.Context) {
 
 // CloseAll stops all subsystems and closes the cloud connection.
 func (m *Manager) CloseAll() {
-	ctx, cancelFunc := context.WithTimeout(context.Background(), time.Minute)
+	ctx, cancelFunc := context.WithTimeout(context.Background(), time.Minute*2)
```

Comment:
nit: leave comment as to why we chose the time we did, and maybe make this a constant as well